src/lib/components/Footer.svelte

```diff
@@ -31,15 +34,25 @@
       text-align: center;
     }
 
+    @media (max-width: 768px) {
```

Comment:
`/src/lib/styles/responsive.scss` 等を追加して
```scss
@mixin mobile {
  @media (max-width: 768px) {
    @content;
  }
}
```

↓にしてあげると分かりやすいかも
```suggestion
    @include mobile {
```
※他も同様に